### DISCUSSION TITLE
Status should have different colors

### DISCUSSION THREAD

**Discussion by donmb1** [community/community#9157] [Projects and Issues]
Suggesting to make this easily editable or provide standard colors like:
Todo: grey
In Progress: yellow/orange
Done: Green

It should also be possible to filter out items with a specific status

**Comment by mobile-sergey** [comment]
I use emodji for it:
<img width="648" alt="Снимок экрана 2021-12-21 в 17 28 26" src="https://user-images.githubusercontent.com/1146694/146946239-d7c51889-8cd1-46af-b08b-49ec926807fe.png">
